frontend/pages/hosts/ManageHostsPage/HostTableConfig.tsx

```
@@ -78,7 +78,14 @@ const condenseDeviceUsers = (users: IDeviceUser[]): string[] => {
     : condensed;
 };
 
-const lastSeenTime = (status: string, seenTime: string): string => {
+const lastSeenTime = (
+  status: string,
+  seenTime: string,
+  platform?: string,
```

ESLint

[error] Delete `,`

(prettier/prettier)